ga4gh/backend.py

```diff
@@ -18,14 +20,34 @@ class Backend(object):
     """
     def __init__(self, dataDir, variantSetClass):
         self._dataDir = dataDir
+        # Variants
         self._variantSetIdMap = {}
-        # All directories in datadir are assumed to correspond to VariantSets.
-        for variantSetId in os.listdir(self._dataDir):
-            relativePath = os.path.join(self._dataDir, variantSetId)
+        variantSetDir = os.path.join(self._dataDir, "variants")
```

Comment:
I'm deliberately not trying to be tidy and refactor things nicely, as this code should be very short lived. Even a cleaned up and refactored version of this code is very ugly, and I'd rather it be obviously very ugly! I should add a TODO to this effect though.